ledger/participant-integration-api/src/main/scala/platform/store/dao/ParametersTable.scala

```diff
@@ -46,6 +46,16 @@ private[dao] object ParametersTable {
 
   private val SelectLedgerEnd: SimpleSql[Row] = SQL"select #$LedgerEndColumnName from #$TableName"
 
+  // TODO use the parameters.ledger_end_sequential_id once available with the append-only schema
+  private val SelectLedgerEndSequentialId: SimpleSql[Row] =
+    SQL"""
+      SELECT events.event_sequential_id
+      FROM participant_events events INNER JOIN parameters params
+      ON events.event_offset = params.ledger_end
```

Comment:
will this work if the ledger_end is set to party update, or package upload, or transaction rejection etc?

Reply:
This method is only for testing purposes. It will never be used in production until the append-only schema lands.